### DISCUSSION TITLE
Custom New Window - uri about:blank navigate to blank page

### DISCUSSION THREAD

**Discussion by mukeshmishra996** [MicrosoftEdge/WebView2Feedback#2966] [Q&A]
Hi Team,
I am trying to customize the WebView2 popup window. But facing a challenge. When URI is in the form of "about:blank". Page redirected to blank page. Page content not loading. Can any one help me on this.

**Comment by darbid** [comment]
You might find you cannot just use the uri property of the newwindowrequest but the new window generated by webview2. 

Does this question help https://github.com/MicrosoftEdge/WebView2Feedback/issues/2710

**Reply by mukeshmishra996** [reply]
Hi darbid, Thanks for quick response. The above link will help if content of the page is static. But my content is dynamic. When I click on any link on web page and if its opening in new window. Inside of newwindowrequest for e.uri property some time I am getting about:blank value and whenever I assign about:blank value to my custom window webview2 instance. Its redirecting to a blank page.

**Reply by darbid** [reply]
Got an example link or links?

**Reply by mukeshmishra996** [reply]
1. https://stackoverflow.com/questions/69365891/webview2-newwindowrequested-overwriting-existing-window
2. https://stackoverflow.com/questions/64332221/access-webview2-newly-opened-window

**Reply by darbid** [reply]
I meant an example of the website that opens the window in the circumstances you describe.

**Reply by mukeshmishra996** [reply]
@darbid I am able to fix the child window issue. But facing one more challenges. When I assign custom popup e.NewWindow = WebView2Child.CoreWebView2. The interop service of webview2 is broken and I unable to communicate between java script and c#. window.chrome.webview is always undefined for me.

**Reply by darbid** [reply]
i am not sure about that one. I am sorry, someone else might pop in and help, I hope.
